pom.xml

```diff
@@ -1228,6 +1228,26 @@
 						</execution>
 					</executions>
 				</plugin>
+				<!--
+					The plugin is defined in the parent pom, to use the right context when running spring-boot:run in a submodule.
+					For example:
+				    mvn -Pdatabase-drivers -pl test -am spring-boot:run
+				-->
+				<plugin>
+					<groupId>org.springframework.boot</groupId>
+					<artifactId>spring-boot-maven-plugin</artifactId>
+					<version>3.4.2</version>
+					<configuration>
+						<skip>true</skip>
+					</configuration>
```

Comment:
Aangezien dit in de pluginManagement zit hoeft dit niet hoor. Het staat alleen als als deze plugin in een module-pom.xml zit.
```suggestion
```